src/full/Agda/Interaction/Options/Base.hs

```diff
@@ -1434,7 +1439,10 @@ standardOptions =
                     , intercalate ", " $ map fst allHelpTopics
                     ]
 
-    , Option []     ["print-agda-dir"] (NoArg printAgdaDirFlag)
+    , Option []     ["print-agda-data-dir"] (NoArg printAgdaDataDirFlag)
```

Comment:
Since we're quite late in the release cycle for 2.6.4, I'd like to avoid introducing any breaking changes. Can you keep `--print-agda-dir`? Adding a less ambiguous phrasing is fine by me.

Reply:
```
> cabal run agda -- --print-agda-data-dir
/home/pounce/programming/agda/./src/data

> cabal run agda -- --print-agda-dir
/home/pounce/programming/agda/./src/data
```